docs/development.md

```diff
@@ -169,38 +169,41 @@ export GO111MODULE=on
 A pattern that comes up quite often is that an object needs to be updated if it already exists or created if it doesn't. `controller-runtime` provides the `controller-util` package which has a `CreateOrUpdate` function that can help with that. It takes a skeleton object and a reconcile function that is called in both the create and the update case:
```

Comment:
Seems like this file was changed by accident during rebase, or only partially updated?